### PR TITLE
Draft release notes

### PR DESCRIPTION
**Pull request by csharpfritz**

### CODE REVIEW

File: NuGet.Docs/Release-Notes/NuGet-3.4-RC.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## New Features
+
+* NuGet clients now support gzip content-encoding from repositories
+* Support for PDBs from packages in xproj projects
+* Support for iOS and Android build actions in the contentFiles element
+* Support for the netstandard and netstandardapp framework monikers
+
+## New User Interface Features
+
+* Significant performance improvements especially on the Installed, Updates, and Consolidate tabs
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-3.4-RC.md]: with more to come in RTM

File: NuGet.Docs/Release-Notes/NuGet-3.4-RC.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* Installed and Updates tabs are now sorted alphabetically
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-3.4-RC.md]: this is in RTM only

File: NuGet.Docs/Release-Notes/NuGet-3.4-RC.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* Added a Refresh button that allows a search to be refreshed
+* 'All' package sources search has been reintroduced with improved support for slower package repositories
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-3.4-RC.md]: All of the stuff here is RTM only. 

All sources should get a higher visibility on top

**Comment by csharpfritz** [comment]
The Github display of your comments has not made it clear which elements we should list as RTM only.

I have pushed the All resources to the top of the UI features and added a sentence in the intro about more coming in RTM

**Comment by yishaigalatzer** [comment]
All sources is not in rc

**Pull request merged**